internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -47,12 +47,15 @@ const (
 	// supported in the cluster. Any others will be ignored.
 	agentServiceConfigName = "agent"
 
-	serviceName              string = "assisted-service"
+	appName                         = "assisted-service"
```

Comment:
:clap: thank you. Naming is hard. I recall on more than one occasion being close to serviceService or something silly like that but `appName` never occurred to me.

Reply:
Yeah, I actually started with two different services and that's why I made this change originally, but then I ended up using the same one with multiple ports but figured this was an improvement either way :laughing: